.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -54,11 +54,18 @@ body:
       options:
         - label: Windows
         - label: macOS
-        - label: Linux
+        - label: Linux (X11)
+        - label: Linux (Wayland)
         - label: BSD-derived
         - label: Other
     validations:
       required: true
+  - type: textarea
```

Very minor, I find adding space between blocks improves readability.

```suggestion
      required: true

  - type: textarea
```